README.md

```diff
@@ -1,53 +1,99 @@
 <h1 align="center">Marker</h1>
 <p align="center">
-    <br/>
-    <img height=200 src="res/logo.svg" alt="Marker logo"/>
+    <img height=200 src="./res/marker-logo-round.svg" alt="Marker logo" />
 </p>
 
 <br/>
 
-[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
-[![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)](https://github.com/rust-marker/marker/actions?query=event%3Apush+workflow%3A%22Rust+(bors)%22)
+[![Crates.io](https://img.shields.io/crates/v/marker_api.svg)](https://crates.io/crates/marker_api)
+<!--
+FIXME(xFrednet): Add license shield, once crates.io also says:
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/marker_api.svg)](#license)
+-->
 
-**An experimental linting interface for Rust. Let's make custom lints a reality!**
+Marker is an experimental code analysis interface, with the goal to create a stable and user-friendly linting framework for the Rust programming language. Creating custom lints should be a straightforward process, and using them should be as simple as declaring a new dependency.
+
+Let's make custom lints and code analysis a reality!
 
 > **Note**
 >
-> The project is in the very early stages of development, some things are still missing and the API is not stable yet.
+> The project is in the early stages of development, some things are still missing and the API is not stable yet.
+>
+> A collection of limitations, goals and features is available below.
+
+## Features
+
+* **Custom Lints**: Marker provides a framework for everyone, to create and provide custom lints. Allowing you to automatically improve the code quality for your project and users of your crate.
+* **User-Friendly Interface**: Marker provides a new subcommand for [Cargo] that does all the heavy lifting for you. [cargo_marker] can automatically set up a driver for linting, compile lint crate and run them on any project.
+* **Driver independent**: Every code analysis requires a driver that parses the code and provides further information. Marker's API is designed to be driver-independent. Allowing it to support future compilers and potentially IDEs, therefore binging custom lints everywhere. (Currently, only [rustc] is supported)
+
+And more to come, see Marker's goals below.
+
+[Cargo]: https://github.com/rust-lang/cargo/
+<!-- FIXME(xFrednet): Update the link to link to `crates.io` once the first version was released-->
+[cargo_marker]: https://github.com/rust-marker/marker/tree/master/cargo-marker
+[rustc]: https://github.com/rust-lang/rustc/
+
+## Goals
+
+* **Stability**: Marker's API design heavily focuses on stability. Any lint, written after version 1.0.0, should compile and continue to work for years to come.
```

Comment:
Do you mean backward compatibility? The 'written after version 1.0.0' part can be made more clear.

Reply:
Do you think the updated version is better? :)